File: src/main/scala/com/github/dapeng/dms/dto/request/PageRequest.scala
package com.github.dapeng.dms.dto.request

/**
  * 分页查询请求包
  *
  * @author <a href=mailto:leihuazhe@gmail.com>maple</a>
  * @since 2018-11-09 11:03 AM
  */
case class PageRequest(

                        /**
                          *
                          * *
                          * 查询的开始序号（序号从零开始）
                          *
                          **/

                        start: Int,

                        /**
                          *
                          * *
                          * 返回记录数
                          *
                          **/

                        limit: Int,

                        /**
                          *
                          * *
                          * 排序的字段
                          *
                          **/

                        sortFields: Option[String] = None
                      )

File: src/main/java/com/github/dapeng/dms/web/vo/request/PageRequest.java
package com.github.dapeng.dms.web.vo.request;

import lombok.Data;

/**
 * @author <a href=mailto:leihuazhe@gmail.com>maple</a>
 * @since 2018-11-09 1:18 PM
 */
public class PageRequest {
    /**
     * *
     * 查询的开始序号（序号从零开始）
     **/

    private Integer start;

    /**
     * *
     * 返回记录数
     **/

    private Integer limit;

    /**
     * *
     * 排序的字段
     **/
    private String sortFields;

    public Integer getStart() {
        return start;
    }

    public Integer getLimit() {
        return limit;
    }

    public String getSortFields() {
        return sortFields;
    }
}
